
Do not create a dead symlink, but instead copy the binary ;-)

  - Rene Rebe <rene@exactcode.de>

--- ./qmake/Makefile.unix.vanilla
+++ ./qmake/Makefile.unix
@@ -42,7 +42,7 @@
 qmake: $(OBJS) $(QOBJS)
 	$(CXX) -o $@ $(OBJS) $(QOBJS) $(LFLAGS)
 	rm -f @BUILD_PATH@/bin/$@
-	ln -s ../qmake/$@ @BUILD_PATH@/bin/$@
+	cp -a ../qmake/$@ @BUILD_PATH@/bin/$@
 
 install: qmake
 	[ -d @QT_INSTALL_BINS@ ] || mkdir -p @QT_INSTALL_BINS@
